.github/workflows/main.yml

```diff
@@ -6,11 +6,9 @@ on:
   push:
     branches:
       - master
-    tags:
-      - v*

```

Comment:
Why do we remove this?

Reply:
I removed this because now we have workflow just for build release and start debug workflow foreach created tags is useless.